src/components/App.js

```diff
@@ -1,8 +1,14 @@
 import React from 'react';
+import { Route, BrowserRouter as Router } from 'react-router-dom';
+import Login from './Login';
+import Landing from './Landing';
 
 const App = () => (
   <div>
-    <h1>Authors Haven</h1>
+    <Router>
```

Comment:
I suggest routes be in a file Routes.js, as the App.js may be our root for redux

Reply:
@ezrogha implemented suggestion